rasa/cdu/command_processor.py

```diff
@@ -343,6 +357,17 @@ def clean_up_commands(
             )
             clean_commands.insert(0, command)
             continue
+        elif isinstance(command, ClarifyCommand):
+            flows = [all_flows.flow_by_id(opt) for opt in command.options]
+            clean_options = [flow.id for flow in flows if flow is not None]
+            if len(clean_options) != len(command.options):
+                structlogger.debug(
+                    "command_executor.altered_command.dropped_clarification_options",
+                    command=command,
+                    original_options=command.options,
+                    cleaned_options=clean_options,
+                )
+            clean_commands.append(ClarifyCommand(clean_options))
```

Comment:
what if the list becomes empty (or is empty to start with)?

Reply:
yes i think that sounds fair :+1:

Reply:
just dropping an empty clarify for now, same as we do for empty start flow